suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -220,7 +220,23 @@ export const getBip43Type = (path: string) => {
     }
 };
 
-export const getAccountTypeName = (path: string) => {
+type getAccountTypeNameProps = {
+    path: string;
+    label?: string;
+    networkType?: string;
+};
+
+export const getAccountTypeName = ({ path, label, networkType }: getAccountTypeNameProps) => {
+    if (label === 'ledger') {
+        return 'TR_ACCOUNT_TYPE_LEDGER';
+    }
+    if (label === 'legacy') {
+        return 'TR_ACCOUNT_TYPE_LEGACY';
+    }
+    if (label === 'normal' && networkType !== 'bitcoin') {
+        return 'TR_ACCOUNT_TYPE_NORMAL';
+    }
```

Comment:
Use switch here too please, it will be more clean. For the normal case you will just add condition inside the case.